src/lib/layers/editable-geojson-layer.js

```diff
@@ -383,21 +464,34 @@ export default class EditableGeoJsonLayer extends EditableLayer {
     dragStartScreenCoords,
     dragStartGroundCoords
   }: Object) {
-    const { selectedFeatures } = this.state;
-    const editHandleInfo = this.getPickedEditHandle(picks);
-
-    if (!selectedFeatures.length) {
-      return;
-    }
+    // alternative method to forcing the user to click to add a new point;
+    // the following allows a drag initiation to create a new point and
+    // drag it without explicitliy selecting it
+    if (!this.getPickedEditHandle(picks) && this.state.hintPoint.length) {
```

Comment:
If you stick with the hint point being an "intermediate edit handle", you shouldn't need to change this code of adding intermediate positions.